armi/reactor/blocks.py

```diff
@@ -1568,7 +1568,7 @@ class during axial expansion.
 
         See Also
         --------
-        armi.reactor.converters.axialExpansionChanger.py::ExpansionData::_setTargetComponents
+        armi.reactor.converters.axialExpansionChanger.expansionData.ExpansionData::_setTargetComponents
```

Comment:
(Soap boxy / not your fault / pedantic nit)

Should we remove references to private methods in docs for public methods?

I understand this was here before and isn't the scope of this PR and we don't need to make this change. More starting to ring the bell of "stop using private methods or make them public and properly tested"

Reply:
That's an interesting point. Yeah, I'm going to do that.

Reply:
Done